manifests/secrets-encryption.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: EncryptionConfig
+resources:
+  - resources:
+      - secrets
+    providers:
+      - secretbox:
+          keys:
+            - name: key1
```

Comment:
How is this key consumed?

Reply:
According to the [documentation](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/#encrypting-your-data) the key should be used by `kube-apiserver` to encrypt the secrets.

Reply:
I have confirmed we can use something better, currently I set it to `secrets_key`. 
Do you have something in mind that would be better ?